scopes/component/component/ui/menu/menu.tsx

```diff
@@ -69,36 +70,42 @@ function VersionRelatedDropdowns({
   const location = useLocation();
   const isNew = component.tags.isEmpty();
   const lanesContext = useLanesContext();
+  const snapResult = useSnaps(component.id);
```

Comment:
don't we have this info on the component itself? do we need to make another gql query for it? can we show some loader (skeleton) while the user waits?

Reply:
we only have the list of tags on the component. 
we are missing a lot of other information for the dropdown,
* List of all snaps
* Information about who made the change (snap/tag) and when that change was made. Currently, it is just a list of version strings. 

Yes makes sense, I will add a loading skeleton to the dropdown

Reply:
@luvkapur - check if you can add this to the component graphql request, at use-component-query.ts`